.github/workflows/label-notifier.yml

```diff
@@ -0,0 +1,20 @@
+name: Notify users based on issue labels
+permissions:
+  issues: read
+
+on:
+  issues:
+      types: [labeled]
+
+jobs:
+  notify:
+    runs-on: tower-cli
+    steps:
+        - uses: jenschelkopf/issue-label-notification-action@1.3
+          with:
+            recipients:
```

Comment:
nit: IMO we should use teams instead of specific user handles. It will make IT happier